aptos-move/block-executor/src/executor.rs

```diff
@@ -1381,7 +1381,7 @@ where
                                 .chain(serialized_groups.into_iter())
                                 .collect(),
                             patched_events,
-                        );
+                        ).map_err(PanicOr::from).map_err(BlockExecutionError::FallbackToSequential)?;
```

Comment:
any idea what's the TODO above this call?

Reply:
that's comment you added, I don't understand whether it is still relevant?

https://github.com/aptos-labs/aptos-core/commit/b3d3dd41fe226ba49497b1b7389553e789e340a1#diff-f1bd27e83d49c6268e702a1a260afae9e0f5f9bf8c5dc6a6fd318b0ab6120245R1292

Reply:
I do add a lot of dead weight comments.
I think back then we had replace for writes but not yet for groups. Now we have everything. should be safe to delete.